FinniversKit/Sources/Components/Infobox/InfoboxSwiftUIView.swift

```diff
@@ -0,0 +1,113 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import SwiftUI
+
+public struct InfoboxSwiftUIView: View {
+    public let style: InfoboxView.Style
+    public var viewModel: InfoboxViewModel?
+    public var onPrimaryButtonTapped: (() -> Void)?
+    public var onSecondaryButtonTapped: (() -> Void)?
+
+    public init(
+        style: InfoboxView.Style,
+        viewModel: InfoboxViewModel? = nil,
+        onPrimaryButtonTapped: (() -> Void)? = nil,
```

Comment:
Tip: If you want to follow the view modifier pattern you can express these as modifier functions instead. Something like:

```
func onSecondaryButton(_ handler: () -> Void) -> Self {
    var infobox = self
    infobox.onSecondaryButtonTapped = handler
    return infobox
}
```

Not really necessary here, but a nice trick if you end up with a huge init parameter list.

Reply:
OK _that_ was a nice trick! Thanks 🙌